### ISSUE TITLE
Commandos 2 Men of Courage (6830)

### ISSUE THREAD

**Issue by Zero86Sk** [ValveSoftware/Proton#818]
This can easily go onto the whitelist. Works as intended.

My Sytem info:
https://gist.github.com/Zero86Sk/d629f657c1ae861c36657cdf238a7765

**Comment by AwesamLinux** [comment]
Distro: Linux Mint 19.1 Tessa (64 bit)
GPU: Vega 56
Driver/LLVM version: Mesa 19.1.0-devel/9.0.0
Kernel version: 5.1.1-050101-lowlatency
Proton version: 4.2-3

Game is playable, but cut-scene videos are skipped. Also the game launches behind the desktops taskbar like this:
![Screenshot from 2019-05-12 19-59-02](https://user-images.githubusercontent.com/43387963/57586165-2ef18a80-74f2-11e9-9b97-29711fac0414.jpg)

Need to alt+tab between desktop and the game to fix it.

[media_info.txt](https://github.com/ValveSoftware/Proton/files/3170581/media_info.txt)
[steam-6830.log](https://github.com/ValveSoftware/Proton/files/3170582/steam-6830.log)
[system_info.txt](https://github.com/ValveSoftware/Proton/files/3170583/system_info.txt)

In the log I see
`19994.801:0027:0028:fixme:quartz:AsyncReader_QueryInterface No interface for {2dd74950-a890-11d1-abe8-00a0c905f375}!`

#1464 The game uses quartz and the videos (stored as .pop files) are: WMV, VC-1 + WMA

**Comment by Steap** [comment]
I disagree: the game randomly crashes early in the first tutorial mission.

[My system info](https://gist.github.com/Steap/175e51773a0cca16beb8ef7f40e99166)

[The crash log](https://gist.github.com/Steap/2186b5bd960ef8ae84c7cfa254a97e10)

**Comment by ranplayer** [comment]
So I've tested this game with Proton 9.0-4, 10.0-2 (RC) and Experimental (2025-07-11).  My findings:
- 9.0-4: 
  - it seems to work well (just the videos are skipped)
- 10.0: 
  - videos work, but no sound
  - soft mouse locks during gameplay (sometimes even on the menu)

Proton 10 logs: [steam-6830.tar.gz](https://github.com/user-attachments/files/21217796/steam-6830.tar.gz)

Specs:
- OS: Arch
- Kernel: 6.12.29
- GPU: RX 6700 XT (Mesa 25.0-5)
- CPU: Ryzen 7 5700 G (8c/16t)
- DE: Plasma 6.3.5 (X11)

**Comment by ranplayer** [comment]
Intro logos/videos (`.pop` format)  are rendered upside down with Proton 11.0-2 (still no sound, like in Proton 10), but if you try to skip them with the "Esc" key, the window freezes (not happening with Proton 10). Proton Experimental (2026-09-04) is skipping them.

Proton logs: [steam-6830.tar.gz](https://github.com/user-attachments/files/31918162/steam-6830.tar.gz)

Specs:
- OS: Arch
- Kernel: 6.18.49
- GPU: RX 6700 XT (Mesa 26.1.8)
- CPU: Ryzen 7 5700G (8c/16t)
- DE: Plasma 6.7.4 (wayland)
- Display device: 1440p/120hz
